docs/source/raft_configuration.md

```diff
@@ -231,35 +217,24 @@ channel, and then start the Raft instance for that chain.
 After it has successfully done so, the channel configuration can be updated to
 include the endpoint of the new Raft orderer.
 
-Removing a node from a Raft cluster is done by:
+To remove an ordering node from a channel:
+
+  1. Remove its endpoint from the channel config for the channel.
+  2. Remove its entry (identified by its certificates) from the channel configuration. For more information, check out [Add or remove orderers from existing channels](../create_channel/create_channel_participation.html#add-or-remove-orderers-from existing-channels).
```

Comment:
Not sure if it's just something weird with the doc build or my browser but this link is the only one that doesn't show up correctly formatted in the doc build.

Reply:
Yeah it's a missing dash between "from" and "existing". Whoops!